lib/lab_manager/app/endpoints/computes.rb

```diff
@@ -21,7 +21,12 @@ class Compute < Base
         end
 
         get '/:id' do
-          ::Compute.find(params[:id]).to_json
+          begin
+            ids = params[:id].to_s.split(',').each(&:to_i)
+            ::Compute.find(ids.count == 1 ? params[:id] : ids).to_json
+          rescue
```

Comment:
catch only: ActiveRecord::RecordNotFound: 
and resutrn what activerecord exception say: Couldn't find all Computes with 'id': (1, 10, 1000) (found 1 results, but was looking for 3)
halt 404, { message: e.message }.to_json